tests/files/catdog/train_data.json

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "filename": "cat/cat-1.jpg",
+    "class_probabilities": [0.9, 0.1],
```

Comment:
so the idea is to store a list of probabilities for all the potential classes here, and then have another list that has the relative ordering of the class names (e.g., index 0 = cat, index 1 = dog).

I'm wondering if we should consider that likely have sparse labels (i.e., we'll have a large list of potential classes, with most classes getting a 0 probability).

Is there a case to be made that we should only store the non-zero class labels? Something like,

```
"class_probabilities": {
   "TRIAGE:CAT": 0.9,
   "TRIAGE:DOG":0.1,
}
```

Reply:
What you said would need that extra list with class names and indices. 
It could be added in the future, as the [code](https://github.com/triagemd/ontology/blob/master/ontology/ontology.py#L768) in ontology can give us both formats. The one I used was the easiest one, because it required less processing from my side.

Reply:
Also with that approach we would need to store much more columns, which I am not sure if it would be an issue.